src/ui/ui-storyEditor.js

```diff
@@ -96,25 +115,27 @@ https://raw.githubusercontent.com/fluid-project/sjrk-story-telling/master/LICENS
                 "method": "click",
                 "args": ["{that}.events.onEditorPreviousRequested.fire"]
             },
-            "onEditorNextRequested.manageVisibility": {
-                funcName: "sjrk.storyTelling.ui.manageVisibility",
-                args: [
-                    ["{that}.dom.storyEditorPage1"],
-                    ["{that}.dom.storyEditorPage2"],
-                    "{that}.events.onVisibilityChanged"
-                ]
-            },
-            "onEditorPreviousRequested.manageVisibility": {
-                funcName: "sjrk.storyTelling.ui.manageVisibility",
-                args: [
-                    ["{that}.dom.storyEditorPage2"],
-                    ["{that}.dom.storyEditorPage1"],
-                    "{that}.events.onVisibilityChanged"
-                ]
-            },
             "onRemoveBlocksRequested.removeSelectedBlocks": {
                 funcName: "sjrk.storyTelling.ui.storyEditor.removeSelectedBlocks",
                 args: ["{that}", "{that}.blockManager.managedViewComponentRegistry"]
+            },
+            "onEditorNextRequested": {
```

Comment:
For consistency, include the namespace with the event as above. 

The same comment applies to the listener at line 126.